deployment/docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+version: '3.8'
+
+services:
+  cordra:
+    build:
+      context: cordra
+      dockerfile: docker/Dockerfile
+    env_file:
+      - variables.env
+  tettris_marketplace:
+    container_name: tettris_marketplace
+    restart: always
+    image: public.ecr.aws/dissco/tettris-marketplace:latest
+  nginx:
+    container_name: nginx
+    restart: unless-stopped
+    image: nginx:1.25
```

Comment:
Can we move to the latest version, 1.26

Reply:
Already did that on the server but forgot to update the deployment file, so updated the deployment file